eclair-core/src/test/scala/fr/acinq/eclair/payment/Bolt12InvoiceSpec.scala

```diff
@@ -78,7 +78,7 @@ class Bolt12InvoiceSpec extends AnyFunSuite {
     val requestWithUnknownTlv = basicRequest.copy(records = TlvStream(basicRequest.records.records, Set(GenericTlv(UInt64(87), hex"0404"))))
     val invoice = Bolt12Invoice(requestWithUnknownTlv, randomBytes32(), nodeKey, 300 seconds, Features.empty, Seq(createPaymentBlindedRoute(nodeKey.publicKey)))
     assert(invoice.records.unknown == Set(GenericTlv(UInt64(87), hex"0404")))
-    assert(invoice.validateFor(requestWithUnknownTlv).isRight)
+    assert(invoice.validateFor(requestWithUnknownTlv, nodeKey.publicKey).isRight)
```

Comment:
Can you add a test validating an invoice created with a blinded `node_id`?

Reply:
Done